actionpack/lib/action_dispatch/middleware/stack.rb

```diff
@@ -133,7 +133,7 @@ def delete(target)
       middlewares.reject! { |m| m.name == target.name }
     end
 
-    def remove(target)
+    def delete!(target)
```

Comment:
Could you also add some rdocs for `delete!` and `delete`?

Reply:
Sure I will! 😄